testsuites/store/src/main/java/org/eclipse/rdf4j/sail/InferencingTest.java

```diff
@@ -109,6 +110,7 @@ public void runTest(Sail sailStack, String subdir, String testName, boolean isPo
 					con.rollback();
 				}
 			}
+			e.printStackTrace();
```

Comment:
This is the culprit of the compilation failure: this e.printStacktrace is _outside_ the catch block.